src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1223,6 +1223,25 @@ format layout_optimizer::get_preferred_format(program_node& node) {
         expected = _forcing_map.at(node.id()).first;
     } else if (node.is_type<convolution>()) {
         expected = get_expected_format(node.as<convolution>());
+        // Set expected input and output preferred format for int8 onednn convolution in dynamic.
```

Comment:
please add additional comment: `// This is a temporal implementation because we are guessing required format for OneDNN conv. In the end, we should use byxf format`